packages/scratch-vm/src/extensions/scratch3_musiccreation/index.js

```diff
@@ -50,6 +51,11 @@ class Scratch3MusicCreation {
                     {text: "mystery 4", value: 4},
                     {text: "mystery 5", value: 5},
                     {text: "mystery 6", value: 6}];
+        
+        this.displayOptions = [{text: "sheet music", value: 1},
```

Comment:
Let's use strings for values before we figure out this[ json issue](https://github.com/mitmedialab/prg-extension-boilerplate/issues/87) (just in case we can't fix it)